jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/KubernetesHelperTest.java

```diff
@@ -469,6 +470,10 @@ void exportKubernetesClientConfigToFile_whenInvalidFileProvided_thenThrowExcepti
     @Test
     @DisplayName("should work with KubernetesClient config provided by KubernetesMockServer")
     void exportKubernetesClientConfigToFile_worksWithKubernetesMockServer(@TempDir Path temporaryFolder) throws IOException {
+        String expectedHostName = mockServer.getHostName();
+        int expectedPort = mockServer.getPort();
```

Comment:
Instead of creating intermediate variables, could you please inline these declarations?